src/RegistryCoordinator.sol

```diff
@@ -295,51 +267,72 @@ contract RegistryCoordinator is
     }
 
     /**
-     * @notice Updates the stakes of all operators for each of the specified quorums in the StakeRegistry. Each quorum also
-     * has their quorumUpdateBlockNumber updated. which is meant to keep track of when operators were last all updated at once.
-     * @param operatorsPerQuorum is an array of arrays of operators to update for each quorum. Note that each nested array
-     * of operators must be sorted in ascending address order to ensure that all operators in the quorum are updated
-     * @param quorumNumbers is an array of quorum numbers to update
-     * @dev This method is used to update the stakes of all operators in a quorum at once, rather than individually. Performs
-     * sanitization checks on the input array lengths, quorumNumbers existing, and that quorumNumbers are ordered. Function must
-     * also not be paused by the PAUSED_UPDATE_OPERATOR flag.
+     * @notice For each quorum in `quorumNumbers`, updates the StakeRegistry's view of ALL its registered operators' stakes.
+     * Each quorum's `quorumUpdateBlockNumber` is also updated, which tracks the most recent block number when ALL registered
+     * operators were updated.
+     * @dev stakes are queryed from the Eigenlayer core DelegationManager contract
+     * @param operatorsPerQuorum for each quorum in `quorumNumbers`, this has a corresponding list of operators to update.
+     * @dev Each list of operator addresses MUST be sorted in ascending order
+     * @dev Each list of operator addresses MUST represent the entire list of registered operators for the corresponding quorum
```

Comment:
might not hurt to acknowledge the potential race condition here? (i.e. if an operator registers/deregisters, it can cause this to revert)